### PR TITLE
Implemented feedback, clarified language, added item ID

### PR DESCRIPTION
**Pull request by miriamseely**

### CODE REVIEW

**Review by steveoh — COMMENTED** [review]
looks good!

**Review by jacobdadams — APPROVED** [review]

File: metadata/demographic/census_blocks_2020.md [hunk 1]
```diff
@@ -1,90 +1,77 @@
...
 ## Secondary Category
 
-<!-- Another category the layer fits in, if applicable. Can be left blank. -->
-
 ## Data Page Link
 
-<!-- Link to the layer's data page on gis.utah.gov -->
 https://gis.utah.gov/products/sgid/demographic/census-2020-blocks/
 
 ## Update
```
Comment (steveoh) [inline metadata/demographic/census_blocks_2020.md:69]: @jacobdadams if there not going to be text under this heading, we should probably remove it from the template

Comment (jacobdadams) [reply metadata/demographic/census_blocks_2020.md:69]: Yeah, I guess that's just in there for hierarchy's sake. Should we do the same with the description heading, and move all the level 3 items to level 2

Comment (steveoh) [reply metadata/demographic/census_blocks_2020.md:69]: yeah that makes sense to me. it will make things a little easier in the future

**Pull request merged**